examples/utils_glue.py

```diff
@@ -388,6 +388,37 @@ def _create_examples(self, lines, set_type):
         return examples
 
 
+class PricePredictionProcessor(DataProcessor):
+    """Processor for price prediction dataset."""
+
+    def get_train_examples(self, data_dir):
+        """See base class."""
+        return self._create_examples(
+            self._read_tsv(os.path.join(data_dir, "train.tsv")), "train")
+
+    def get_dev_examples(self, data_dir):
+        """See base class."""
+        return self._create_examples(
+            self._read_tsv(os.path.join(data_dir, "dev.tsv")), "dev")
+
+    def get_labels(self):
+        """See base class."""
+        return [None]
+
+    def _create_examples(self, lines, set_type):
+        """Creates examples for the training and dev sets."""
+        examples = []
+        for (i, line) in enumerate(lines):
+            if i == 0:
+                continue
+            guid = "%s-%s" % (set_type, line[0])
+            text_a = line[1]
```

Comment:
Each line just needs to have an ID, the text, and the price label value, tab-separated.